ledger-core/rms/proto_vnvn.proto

```diff
@@ -35,3 +35,53 @@ message MessageExample {
         Binary Str = 40;
     }
 }
+
+message VObjectTranscriptReport {
+    option (insproto.id) = 1015;
+
+    //*** Start of predefined section ***//
+    reserved 1 to 18, 20 to 1799;
+    // FIXME: this is here just to make generator silent
+    Any Record = 19 [(gogoproto.embed) = true];
+    //*** End of predefined section ***//
+
+    uint32 AsOf = 1800 [(gogoproto.casttype) = "PulseNumber"];
+    Reference Object = 1801;
+
+    message TranscriptEntryIncomingRequest {
+        option (insproto.id) = 0;
```

Comment:
Are you sure what we need to use zero value?

Reply:
we will figure out while moving forward. It's sub-part of a message. Possible that we don't need id at all.